src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -350,14 +350,45 @@ def switch_browser(self, index_or_alias):
         self.debug('Switched to browser with Selenium session id %s.'
                    % self.driver.session_id)
 
+    @keyword
+    def get_browser_ids(self):
```

Comment:
I did not understand where this keyword would be useful? Let's discuss.

Reply:
The idea of this PR was to resolve complexity of window and browser handling.
1st reason for `get browser ids` and aliases was that we have corresponding Keywords for windows. This is for completeness.

2nd reason: It can happen, that browsers are opened in a very high level Suite Setup. Or that multiple browsers are open. If you want to Switch to a specific browser that has a specific site, you had no possibility to go through all Browsers. You could just iterate over some numbers, ` in range 0-10` but if some browsers are allready closed you will get problems.

Reply:
I see your point. Then there should be documentation for the users how the keyword can be used with other keywords. There could be also tests to cover the usage scenario, so that we do not accidentally break the feature. 

Same comment goes with the `get_browser_aliases` comment.

Reply:
It is hard for me to make an understandable pice of robot code.

Would be something like nested for loops. over browsers and windows.
And this is only possible with keywords in Keywords in robot.
Which makes it hard do document in Table.

Usecase would be switch to a window by its sontent. 
Of course i could do something like this:
```
Logging Window Titles for Each Browser
    @{browser_ids}=    Get Browser Ids
    FOR    ${id}    IN    @{browser_ids}
        @{window_titles}=    Get Window Titles    browser=${id}
        Log    Browser ${id} has these windows: ${window_titles}
    END
```

I could do this into the doc...
But it would not work as atest, because old Robot Version does not support this loop.

Reply:
another example of how `Get Browser Ids` could be usefull:

```
*** Keywords ***
Switch Window By Location
    [Arguments]    ${selected_location}
    @{IDs}=    Get Browser Ids
    FOR     ${id}    IN    @{IDs}
        @{locations}=    Get Locations    browser=${id}   
        Run Keyword If    '${selected_location}' in $locations
        ...    Switch Window    url:${selected_location}    browser=${id}
    END
```